network.tf

```diff
@@ -37,18 +37,18 @@ resource "azurerm_virtual_hub_connection" "hub_connection" {
 
 # Routing intent
 resource "azurerm_virtual_hub_routing_intent" "routing_intent" {
```

Comment:
Could the `azurerm_virtual_hub_routing_intent` be implemented as a separate sub-module, or to nest it within the hub sub-module since this functionality operates at the individual hub level?

friendly reminder to include the moved block if it has been decided to move it as suggested above

Reply:
Yes. Same explanation as vnet-hub-conn

Reply:
This feature can be added in the next phase.

Reply:
We will add it in the next phase for sure.